### DISCUSSION TITLE
Unable to load env in deployed version

### DISCUSSION THREAD

**Discussion by Garry028** [electron-vite/electron-vite-vue#480] [Q&A]
I have deployed the electron vue for now on amplify but I'm unable to load env they're coming as undefined in the console after deployment. they're coming correctly in local environment  as I'm console logged them.
In amplify I'm getting a warning **2024-04-04T15:15:45.155Z [WARNING]: !Failed to set up process.env.secrets**
RENDERER_VITE_APP_    this is the prefix I have used. by reading the documentation

**Comment by caoxiemeihao** [comment]
The default prefix is `VITE_`
https://vitejs.dev/config/shared-options.html#envprefix
<img width="309" alt="image" src="https://github.com/electron-vite/electron-vite-vue/assets/26263658/a3bafaf7-b359-401b-81f4-6e880fee2102">

**Reply by Garry028** [reply]
but as per electron vite documentation "RENDERER_VITE_APP_" this is correct prefix
